interpreter/core/respond.py

```diff
@@ -112,9 +112,12 @@ def respond(interpreter):
                     interpreter.messages[-1]["language"] = "shell"
 
                 # Get a code interpreter to run it
-                language = interpreter.messages[-1]["language"]
+                language = interpreter.messages[-1]["language"].replace(" ", "")
                 if language not in interpreter._code_interpreters:
-                    interpreter._code_interpreters[language] = create_code_interpreter(language)
+                    try:
+                        interpreter._code_interpreters[language] = create_code_interpreter(language)
+                    except:
+                        print(f"Error: tried to create a code_interpreter with: '{language}'")
```

Comment:
Should we consider checking against the available code interpreters in the [`language_map`](https://github.com/KillianLucas/open-interpreter/blob/main/interpreter/code_interpreters/language_map.py#L10) rather than trying to create an interpreter that might fail?